### PR TITLE
North West 4 - Morteza Khojasteh - JavaScript Core 1 - Week 3 (exercises)

### PR DESCRIPTION
**Pull request by Morteza-Khojasteh**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Review by xtremelibasic — COMMENTED** [review]
Nice work!

**Comment by myaskylar** [comment]
nice and clean code Morteza. Great Job!

File: 1-exercises/F-array-forEach/exercise.js [hunk 1]
```diff
@@ -7,8 +7,19 @@
...
+  if (num % 3 == 0 && num % 5 == 0) {
+    console.log("FizzBuzz");
+  } else if (num % 3 == 0) {
+    console.log("Fizz");
+  } else if (num % 5 == 0) {
+    console.log("Buzz");
+  } else {
+    console.log(num);
+  }
+});
```
Comment (myaskylar) [inline 1-exercises/F-array-forEach/exercise.js:22]: Very nicely write your code here. Great job!

**Comment by Morteza-Khojasteh** [comment]
> Nice work!

Thanks Amanda

**Comment by Morteza-Khojasteh** [comment]
> nice and clean code Morteza. Great Job!

Thanks Mya

File: 1-exercises/C-array-every/exercise.js [hunk 2]
```diff
@@ -2,10 +2,12 @@
    This program should check if the array `group` contains only students
 */
 
-var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
-var group = ["Austine", "Dany", "Swathi", "Daniel"];
+const students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
+const group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+const groupIsOnlyStudents = group.every((element) =>
```
Comment (lannem) [inline 1-exercises/C-array-every/exercise.js:8]: nice use of an arrow function :)

File: 1-exercises/E-array-map/exercise.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 // Using the .map() method, create a new array with `numbers` multiplied by 100
 // Write multiple solutions using different syntax (as shown in the README)
 
-var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
+const numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
+const numbersTimes100 = numbers.map( element => element * 100 );
+console.log(numbersTimes100);
```
Comment (lannem) [inline 1-exercises/E-array-map/exercise.js:6]: this is good but the instructions asked you to write the `map()` function in multiple ways :)

File: 1-exercises/F-array-forEach/exercise.js [hunk 4]
```diff
@@ -7,8 +7,19 @@
   An array with numbers 1-15 has been provided.
 */
 
-var arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+const arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach((num) => {
```
Comment (lannem) [inline 1-exercises/F-array-forEach/exercise.js:12]: nice and neat well done :)

**Review by lannem — COMMENTED** [review]
Well done Morteza :)

File: 1-exercises/C-array-every/exercise.js (hunk 2 above)
Comment (Morteza-Khojasteh) [reply 1-exercises/C-array-every/exercise.js:8]: Thanks

File: 1-exercises/E-array-map/exercise.js (hunk 3 above)
Comment (Morteza-Khojasteh) [reply 1-exercises/E-array-map/exercise.js:6]: Yeah, I didn't have enough time :)

File: 1-exercises/F-array-forEach/exercise.js (hunk 4 above)
Comment (Morteza-Khojasteh) [reply 1-exercises/F-array-forEach/exercise.js:12]: Thank you so much!

**Pull request closed without merging**